production/helm/loki/templates/querier/deployment-querier.yaml

```diff
@@ -104,13 +104,13 @@ spec:
           {{- end }}
           securityContext:
             {{- toYaml .Values.loki.containerSecurityContext | nindent 12 }}
-          {{- with .Values.loki.livenessProbe }}
+          {{- if .Values.loki.livenessProbe }}
           livenessProbe:
-            {{- toYaml . | nindent 12 }}
+            {{- mergeOverwrite (deepCopy (default dict .Values.loki.livenessProbe)) (default dict .Values.querier.livenessProbe) | toYaml | nindent 12 }}
```

Comment:
I would like to see this pattern below, since we use this already on other parts of the chart.

```
          {{- with .Values.querier.livenessProbe | default .Values.loki.livenessProbe }}
          livenessProbe:
            {{- . | toYaml | nindent 12 }}
```

Reply:
It seems 15s is the default, except for the helm chart.

We can use the values from the operator as default:

https://github.com/grafana/loki/blob/04f89298c461dda025976419e7501c97870f6a2e/operator/internal/manifests/var.go#L581-L612

But please, open a distinct PR for that.

Reply:
👍 This will be a good change to put in place and then can work on getting more fine grained control

Reply:
@jkroepke Created a separate MR https://github.com/grafana/loki/pull/19090 to set the default readiness probe values